index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World, we are software developers!</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <div id="main">
+
+        <section id="heading">
+            <h1 id="title">Dr. Martin Luther King</h1>
```

Comment:
Please use a class selector because “id” is unique on a page and can only apply to at most one element, while a “class” selector can apply to multiple elements.